en-US/about/project/license.md

```diff
@@ -61,6 +61,42 @@ To read the [LGPL](http://www.gnu.org/licenses/lgpl.html) and [GPL](http://www.g
+
+##Translators
+Rafael Ferran i Peralta - ca - Catalan
+Philipp Poll - de - Deutsch
+Apostolos Syropoulos - el - Greek
+George Williams - en_GB
+Walter Echarri - es - Spanish
+Pierre Hanser - fr - French
+Yannis Haralambous - fr - French
+Claudio Beccari - it - Italian
+KANOU Hiroki - ja - Japanese
+TANIGAWA Takashi - ja - Japanese
+OKANO Takayoshi, ja - Japanese
+Hiran - ml - Malayalam
+Michal Nowakowski - po - Polish
+Jose Da Silva - pt - Portuguese
+Alexandre Prokoudine - ru - Russian
+Valek Filippov - ru - Russian
+Yuri Chornoivan - uk - Ukrainian
+Clytie Siddall - vi - Vietnamese
+Lee Chenhwa - zh_CN - Chinese
+Wei-Lun Chao - zh_TW - Traditional Chinese
+
+##Notes
+The general direction FontForge is going in is towards GPLv3+, however, you must NOTE that a large portion of FontForge is still licensed as BSD CODE.
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+
+The name of the author may not be used to endorse or promote products derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE AUTHORS ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
On May 6, 2015 09:08:30 AM Andreas Larsen wrote:
> 
> ?

You need to pay attention to the double quote marks.
If you use a word processor, these tend to add the enhanced version of the 
quote on the end-part. A lot of compilers configurators and parsers are only 
expecting the simpler double quote mark, so please use the simpler version.